test/lib/ansible_test/_internal/debugging.py

```diff
@@ -451,4 +469,5 @@ def detect_debugpy_options() -> DebugpyOptions | None:
     return DebugpyOptions(
         port=port,
         adapter_access_token=opts.adapter_access_token,
+        debugpy_package_path=str(pathlib.Path(debugpy.__file__).resolve().parent.parent.absolute()),
```

Comment:
`.absolute()` isn't really needed after `.resolve()`. The latter would already make it absolute and accessing parents would respect that. In fact, `.absolute()` has this confusing semantics that makes people thing that it somehow resolves paths correctly but it just prepends `/` mostly, IIRC.
```suggestion
        debugpy_package_path=str(pathlib.Path(debugpy.__file__).resolve().parent.parent),
```